cylc/rose/entry_points.py

```diff
@@ -217,3 +219,28 @@ def record_cylc_install_options(
     )
     dumper(rose_suite_conf, rose_conf_filepath)
     return cli_config, rose_suite_conf
+
+
+def copy_config_file(
+    dest_root=None,
+    opts=None,
+    dir_=None,
+):
+    if (
+        dest_root is None or
+        dir_ is None
+    ):
+        return False
+
+    rundir = Path(dest_root)
+    srcdir = Path(dir_)
+    srcdir_rose_conf = srcdir / 'rose-suite.conf'
+    rundir_rose_conf = rundir / 'rose-suite.conf'
+
+    if not srcdir_rose_conf.is_file():
```

Comment:
One thing to bear in mind, `.is_file()` returns `True` if it's a symlink. I guess it's probably pretty unlikely that someone would make `rose-suite.conf` a symlink, but possible?

Reply:
If it is then we end up copying the target using `shutil.copy2(srcdir_rose_conf, rundir_rose_conf)`, which seems to me like reasonable behaviour.